.travis.yml

```diff
@@ -56,6 +57,10 @@ matrix:
     - os: linux
       env: TARGETS="TEST_ENVIRONMENT=0 -C metricbeat testsuite"
     - os: osx
+      env: TARGETS="-C filebeat testsuite"
```

Comment:
Perhaps we could switch to `include` instead of exclude in the future as the exclude becomes larger and larger: https://docs.travis-ci.com/user/customizing-the-build#Explicitly-Including-Jobs This should also make it easier to read.

We can try this in a future PR.